mmv1/products/notebooks/api.yaml

```
@@ -810,10 +818,17 @@ objects:
                 description: |
                   The type of vNIC to be used on this interface. This may be gVNIC
                   or VirtioNet.
+                input: true
                 values:
                   - UNSPECIFIED_NIC_TYPE
                   - VIRTIO_NET
                   - GVNIC
+              - !ruby/object:Api::Type::String
+                name: "reservedIpRange"
+                description: |
+                  Optional. Reserved IP Range name is used for VPC Peering. The
```

```suggestion
                  Reserved IP Range name is used for VPC Peering. The
```
We can remove `Optional` from the description, as it would auto generated `(optional)` for the field in the downstream documentation if we don't set the field `required` in the yaml file.